stl/inc/ranges

```diff
@@ -2506,6 +2510,31 @@ namespace ranges {
         /* [[no_unique_address]] */ _Vw _Range{};
         range_difference_t<_Vw> _Count = 0;
 
+        _NODISCARD constexpr auto _Find_first() {
+            if constexpr (!sized_range<_Vw>) {
+                return _RANGES next(_RANGES begin(_Range), _Count, _RANGES end(_Range));
+            }
+
+            auto _Offset = _RANGES distance(_Range);
+            if constexpr (bidirectional_range<_Vw> && common_range<_Vw>) {

```

Comment:
I believe this is not desireable as is, as it would also trigger for random_access ranges

Reply:
We don't call this function when `_Vw` is random-access and sized; see 2561. I'll throw in an `_STL_INTERNAL_STATIC_ASSERT` here for clarity.